src/pickers.rs

```diff
@@ -15,6 +15,33 @@ pub trait Picker: std::fmt::Debug + Sync + Send {
     fn pick<'a>(&self, choices: &'a [Choice], scores: &Query<&Score>) -> Option<&'a Choice>;
 }
 
+/**
+Picker that chooses the `Choice` with the highest non-zero [`Score`], and the first highest in case of a tie.
+
+### Example
+
+```no_run
+Thinker::build()
+    .picker(HighestThenFirst)
+    // .when(...)
+```
+ */
+#[derive(Debug, Clone, Default)]
+pub struct HighestThenFirst;
```

Comment:
can we just call this `Highest` and document that "in the case of a tie, the first one will be used"?